dev-docs/modules/reconciliationRtdProvider.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: page_v2
+title: Reconciliation SDK Module
+description: Reconciliation Real Time Data Module
+page_type: module
+module_type: rtd
+module_code: reconciliation
+display_name: Reconciliation
+enable_download: true
+sidebarType: 1
+---
+
+# Reconciliation SDK Module
+{:.no_toc}
+
+* TOC
+{:toc}
+
+## Overview
+
+The purpose of this Real Time Data Provider is to allow publishers to match impressions accross the supply chain.
```

Comment:
Typo -- "accross". Also, this sentence is kinda redundant with the one beneath it. Both start with "The purpose of". Might consider combining them.

Reply:
Thank you.
I've rewritten the description